CNV-55957: Add documentation for the new tree view (#93239)
openshift/openshift-docs: modules/virt-listing-vms-cli.adoc

// Module included in the following assemblies:
//
// * virt/managing_vms/virt-list-vms.adoc

:_mod-docs-content-type: PROCEDURE

[id="virt-listing-vms-cli_{context}"]
= Listing virtual machines by using the CLI

> You can list all virtual machines (VMs) in your cluster or limit the list to VMs in a certain namespace by using the {oc-first}.

Reviewer: ```suggestion
You can either list all of the virtual machines (VMs) in your cluster or limit the list to VMs in a specified namespace by using the {oc-first}.
```

.Prerequisites

* You have installed the {oc-first}.

.Procedure

> * To list all VMs in your cluster, run the following command:

Reviewer: ```suggestion
* List all of the VMs in your cluster by running the following command:
```
active voice

+
[source,terminal]
----
$ oc get vms -A
----

> * To list VMs in a specific namespace, run the following command:

Reviewer: ```suggestion
* List only the VMs in a specific namespace by running the following command:
```
